backend/app/models/user.rb

```diff
@@ -40,11 +42,11 @@ def drafts
   end
 
   def admin
-    role == 'admin'
+    admin?
   end
 
   def staff
-    role == 'staff'
+    author?
```

Comment:
ok, I just made it through the `web-client` I guess the only use for roles was in `admin` case.